ui/src/app/views/homepage/component.tsx

```diff
@@ -341,72 +287,218 @@ export class HomepageComponent implements OnInit, OnDestroy {
+    } else {
+      this.timer = setTimeout(() => {
+        reload();
+      }, 10000);
     }
-    return completedTasks;
   }
 
-  public get completedTasksAsPercentage() {
-    return this.completedTasks / this.numberOfTotalTasks * 100;
+  openVideo(videoLink: string): void {
+    this.setState({videoOpen: true, videoLink: videoLink});
   }
 
-  // TODO [RW-1887] Dead code, please remove
-  reloadSpinner(): void {
-    this.spinnerValues = [
-      {
-        'name': this.completedTasksName,
-        'value': this.completedTasks
-      },
+
+  render() {
+    const {billingProjectInitialized, videoOpen, accessTasksLoaded,
+        accessTasksRemaining, eraCommonsLinked, eraCommonsError, trainingCompleted,
+        quickTour, videoLink, firstVisit} = this.state;
+    const quickTourResources = [
       {
-        'name': this.unfinishedTasksName,
-        'value': this.numberOfTotalTasks - this.completedTasks
+        src: '/assets/images/QT-thumbnail.svg',
+        onClick: () => this.setState({quickTour: true})
+      }, {
+        src: '/assets/images/cohorts-thumbnail.png',
+        onClick: () => this.openVideo('/assets/videos/Workbench Tutorial - Cohorts.mp4')
+      }, {
+        src: '/assets/images/notebooks-thumbnail.png',
+        onClick: () => this.openVideo('/assets/videos/Workbench Tutorial - Notebooks.mp4')
       }
     ];
-  }
-
-  ngOnDestroy(): void {
-    clearTimeout(this.billingProjectQuery);
-  }
+    const footerLinks = [{
+      title: 'Working Within Researcher Workbench',
+      links: ['Researcher Workbench Mission',
+        'User interface components',
+        'What to do when things go wrong',
+        'Contributing to the Workbench']
+    },
+      {
+        title: 'Workspace',
+        links: ['Workspace interface components',
+          'User interface components',
+          'Collaborating with other researchers',
+          'Sharing and Publishing Workspaces']
+      },
+      {
+        title: 'Working with Notebooks',
+        links: ['Notebook interface components',
+          'Notebooks and data',
+          'Collaborating with other researchers',
+          'Sharing and Publishing Notebooks']
+      }];
 
-  addWorkspace(): void {
-    navigate(['workspaces/build']);
-  }
+    return <React.Fragment>
+      <div style={homepageStyles.backgroundImage}>
+        <div style={{display: 'flex', justifyContent: 'center'}}>
+          <div style={homepageStyles.singleCard}>
+            {accessTasksLoaded ?
+              (accessTasksRemaining ?
+                (<WorkbenchAccessTasks eraCommonsLinked={eraCommonsLinked}
+                                       eraCommonsError={eraCommonsError}
+                                       trainingCompleted={trainingCompleted}/>
+                ) : (
+                  <div>
+                    <div style={{flexDirection: 'row', height: '17.47%', marginBottom: '0.5rem',
+                      justifyContent: 'flex-start', flexWrap: 'nowrap', display: 'flex'}}>
```

Comment:
nit: can this be moved to style tag. It would be better to limit inline styles to 3.

Reply:
will do